uyuni/base/uyuni-base.spec

```diff
@@ -88,29 +94,34 @@ Basic filesystem hierarchy for Uyuni proxy.
 %install
 mkdir -p %{buildroot}/etc/rhn
 mkdir -p %{buildroot}/usr/share/rhn/proxy
-%if ! (0%{?rhel} == 6 || 0%{?suse_version} == 1110)
```

Comment:
uyuni-base-common is used in osad.spec which also build for these old versions.
We have this bad situations that some packages contain client and server components.

So it is better to keep the rhel 6 test here until we really do not build for RHEL 6 clients anymore.
Here and the other 3 places in this file.

Reply:
Done.